### PR TITLE
2015.09: added release notes

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: release-notes.txt [hunk 1]
```diff
@@ -1,3 +1,296 @@
...
+About this release:
+============
+This release introduces the GNRC network stack, a completely new, highly
+modularized and configurable IPv6/6LoWPAN stack. It also includes xtimer as a
+new timer subsystem for accurate short- and long-term timers. Moreover,
+peripheral drivers, board, and CPU support has been tidied up and contains
+about 50% less duplication in the build system.
+
+About 580 pull requests with about 2,500 commits have been merged since the
+last release and 120 additional issues has been solved. 62 people contributed
```
Comment (cgundogan) [inline release-notes.txt]: s/has/have/

File: release-notes.txt [hunk 2]
```diff
@@ -1,3 +1,296 @@
+RIOT-2015.09 - Release Notes
+============================
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things: 8-bit
```
Comment (emmanuelsearch) [inline release-notes.txt]: low-end devices that are typically found in the Internet of Things: devices based on 8-bit

File: release-notes.txt [hunk 3]
```diff
@@ -1,3 +1,296 @@
...
+* improved documentation
+* fixed several IPC message queue initializations
+- removed hwtimer
+
+Drivers
+---------
+* optimized/remodeled GPIO interface
+* optimized/remodeled TIMER interface
+- temporarily removed CC2420 driver (awaiting last bug fixes for a rewrite)
+- re-implementation of the CC110x driver against the peripheral interface
```
Comment (emmanuelsearch) [inline release-notes.txt]: * instead of - here ?

File: release-notes.txt [hunk 4]
```diff
@@ -1,3 +1,296 @@
...
+
+Network Stack
+---------------------
+* temporarily removed and being refactored:
```
Comment (emmanuelsearch) [inline release-notes.txt]: currently being refactored

File: release-notes.txt [hunk 5]
```diff
@@ -1,3 +1,296 @@
...
+Packages
+--------
+* updated CMSIS HAL to version 4.3
+
+Other
+-----
+* clean-up of deprecated system and network libraries
+* clean-up of deprecated boards and drivers
+
+Fixed Issues from the last release
```
Comment (emmanuelsearch) [inline release-notes.txt]: Selected Issues Fixed since the Last Release (more were fixed ;)

**Comment by emmanuelsearch** [comment]
ACK with the minor nits I indicated in the file. Thanks!

**Comment by thomaseichinger** [comment]
\o/

**Pull request merged**